playbooks/bibdata.yml

```
@@ -12,12 +12,31 @@
     - ../group_vars/bibdata/vault.yml
     - ../group_vars/bibdata/{{ runtime_env | default('staging') }}.yml
   roles:
+    # re-add when removed from: ruby_s, deploy_user, rails_app, and samba
+    # - role: common
     - role: deploy_user
-    - { role: roles/sidekiq_worker, when: "'worker' in inventory_hostname" }
-    - role: roles/bibdata
-    - role: roles/bibdata_sqs_poller
-    - role: 'hr_share'
-    - role: roles/datadog
+    - role: redis
+    # re-add when removed from passenger
+    # - role: ruby_s
+    # re-add when removed from rails_app
+    # - role: bind9
+    # re-add when removed from rails_app
```

This is nitpicky, but I found it hard to know at a glance which role each comment applied to.  Would it be possible to add them to the comment?
```suggestion
    # re-add when passenger role is removed from rails_app
```

---

Thanks @sandbergja! Good idea, I've tried to do that